hpi-swt2/lecture-portal
# Feature/#122 file download

# Resolves #122

This PR adds / introduces the feature of downloading an uploaded file 

Acceptance Criteria:
- [x] Clicking a button / following a URL, will start a download of any uploaded file specified 


Testing steps:
- go to /uploaded_files
- upload a file
- there should be a download link now at /uploaded_files
- clicking that link or following URL /uploaded_files/1 should download that file

Steps needed for migration:
- None

Definition Of Done:
